tutorials/introductory/getting_started.py

```diff
@@ -0,0 +1,846 @@
+# |   allows for robust control and is | - The module ``pyplot`` operates   |
+# |   useful for generalized code.     |   similarly to MATLAB and is       |
+# |                                    |   convenient for interactive       |
+# |                                    |   environments.                    |
+# +------------------------------------+------------------------------------+
+#
+# .. note::
+#
+#     The Matplotlib community does not recommend interchanging explicit and
+#     implicit strategies. When using one as standard, all code following
+#     the same strategy reduces troubleshooting issues. Switching back and
+#     forth between explicit and implicit programming may yield errors.
+#
+# For other techniques of creating plots with Matplotlib, refer to
+# :ref:`user_interfaces`.
+#
+# Data
+# ----
+#
+# The Matplotlib library manages data in the form of iterables and/or
+# sequenced items. These also take the form of NumPy arrays like
+# `numpy.array` or `numpy.ma.masked_array`. All plotting functions take these
+# data structures.
+#
+
+# Sample Data for Personal Financial Tracking in 2009 & 2010
+
+months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+          'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+income = [950, 950, 950, 950, 950, 950,
+          950, 950, 950, 950, 950, 950]
+chk_acct_09 = [1250, 1325, 1200, 1220, 1100, 1055,
+               1255, 1090, 1190, 1205, 1205, 1180]
+svg_acct_09 = [1000, 1050, 1100, 1150, 1200, 1250,
+               1300, 1350, 1400, 1450, 1500, 1550]
+chk_acct_10 = [1180, 1270, 1280, 1280, 1260, 1140,
+               1270, 1160, 1120, 1250, 1270, 1160]
+svg_acct_10 = [1550, 1600, 1650, 1700, 1750, 1800,
+               1850, 1900, 1950, 2000, 2050, 2100]
+
+##############################################################################
+#
+# .. note::
+#
+#    Other containers, such as data objects from various libraries, may not
+#    work as intended.
+#
+# Explicit and implicit examples of a basic line plot are below. Both of the
+# following plots are identical. Each uses a different approach to graph the
+# data. The results do not change for either approach when using the same data
+# points.
+#
+# Explicit: Object Oriented Programming (OOP)
+# --------------------------------------------
+#
+# Explicit programming for Matplotlib involves using a single instance of the
+# ``pyplot`` module. This unpacks a set of an explicit Figure and Axes.
+# More than one Axes is configurable; however, each Axes only corresponds to a
+# single Figure.
+#
+# Each Axes has its own methods to graph data. In addition, each Axes
+# also uses separate methods to create and manage parts of a Figure. These
+# methods are different from those of the implicit programming approach.
+
+# Explicit programming with OOP
+
+x = months
+y1 = income
+y2 = chk_acct_09
+y3 = svg_acct_09
+y4 = chk_acct_10
+y5 = svg_acct_10
+# Assigning sample data to labeled variables.
```

Comment:
Is there a general consensus to move all comments to the same line on the right for all code? @tacaswell @story645 @timhoffm 

This would also be helpful to include in the Style Guide if this is the standard.

Reply:
I agree with @QuLogic  here.